preload/dynamic_adding_preload.html

```diff
@@ -7,13 +7,17 @@
 <body>
 <script>
     var link = document.createElement("link");
+    if(!link.rellist || !link.rellist.supports("preload")) {
+        assert_unreached("rellist is not defined or browser doesn't support preload.");
+        t.done();
+    }
     link.as = "script";
     link.rel = "preload";
     link.href = "resources/dummy.js";
-    link.onload = t.step_func(function() {
+    link.onload = t.step_timeout(function() {
```

Comment:
There's no spec guaranty that the PerformanceEntry is added to the timeline before the resource's onload handler runs